<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::PhoneCallingView</class>
 <widget class="QWidget" name="Qpid::PhoneCallingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voice Easy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widgetCalling {
	border-image: url(:/images/ui/call_bg.jpg);
}

#labelCallerPhoto,#labelTransPhoto,#labelCalleePhoto{
	min-width: 90px;
	min-height: 90px;
	max-width: 90px;
	min-height: 90px;
}

#labelTransConnect,#labelCalleeConnect{
	min-width: 101px;
	min-height: 17px;
	max-width: 101px;
	min-height: 17px;
}

#labelCallerPhoto,#labelCalleePhoto {
	border-image: url(:/images/ui/default_photo.jpg);
}

#labelTransPhoto {
	border-image: url(:/images/ui/live_interpreter.jpg);
}

#labelCallStatus {
	min-height: 120px;
	padding-bottom: 17px;
	font-size: 18px;
	font-weight: bold;
	color: #ffffff;
}

#labelCallDuration {
	font-size: 18px;
	font-weight: bold;
	color: #ffffff;
}

#pushButtonMaximize {
	border-image: url(:/images/ui/calling_maximize.png);
	min-width: 19px;
	max-width: 19px;
	min-height: 14px;
	max-height: 14px;
}

#labelTransConnect,#labelCalleeConnect {
	border-image: url(:/images/ui/call_connected.png);
}

#labelCallerName,#labelTransName,#labelCalleeName {
	font-size: 12px;
	font-weight: bold;
	color: #ffffff;
	max-width: 90px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetCalling" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frameVerticalCenter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="frameHorizontalCenter">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frameCallPhotoContainer">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QFrame" name="frameCallerPanel">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="labelCallerPhoto">
                    <property name="maximumSize">
                     <size>
                      <width>90</width>
                      <height>90</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="scaledContents">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Qpid::ElidedLabel" name="labelCallerName">
                    <property name="text">
                     <string>You</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelTransConnect">
                 <property name="minimumSize">
                  <size>
                   <width>101</width>
                   <height>17</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>101</width>
                   <height>17</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frameTransPanel">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="labelTransPhoto">
                    <property name="maximumSize">
                     <size>
                      <width>90</width>
                      <height>90</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="scaledContents">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Qpid::ElidedLabel" name="labelTransName">
                    <property name="text">
                     <string>Voice Easy interpreter</string>
                    </property>
                    <property name="scaledContents">
                     <bool>false</bool>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCalleeConnect">
                 <property name="minimumSize">
                  <size>
                   <width>101</width>
                   <height>17</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>101</width>
                   <height>17</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frameCalleePanel">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="labelCalleePhoto">
                    <property name="maximumSize">
                     <size>
                      <width>90</width>
                      <height>90</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="scaledContents">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Qpid::ElidedLabel" name="labelCalleeName">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="scaledContents">
                     <bool>false</bool>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelCallStatus">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widgetCallContainer" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="labelCallDuration">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Qpid::CallControlPanel" name="callControlPanel" native="true"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonMaximize">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qpid::CallControlPanel</class>
   <extends>QWidget</extends>
   <header>callcontrolpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Qpid::ElidedLabel</class>
   <extends>QLabel</extends>
   <header>elidedlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections/>
</ui>
